tests/test_solid_figures.py

```diff
@@ -0,0 +1,100 @@
+import pytest
+from converter import solid_figures
+
+
+_Json_with_figures = {
```

Comment:
What do you think of moving such JSON to a `tests/res` folder?
Imagine that in some distant future format of JSON files on the frontend would change.
Replacing the content of `tests/res` with a new version would be quite straightforward in that case.
Also single JSON file could be reused among many tests

Reply:
I think for now it can stay like this. If we need those constants anywhere else we can move them.  I'm also planning on reviewing the way i provide data to all the tests after i finish the sh12a input part of the converter.

Reply:
ok, so please create an issue for that